src/spikeinterface/preprocessing/tests/test_unsigned_to_signed.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from pathlib import Path
+import shutil
+
+from spikeinterface import set_global_tmp_folder, NumpyRecording
+from spikeinterface.core import generate_recording
+
+from spikeinterface.preprocessing import unsigned_to_signed
+
+import numpy as np
+
+
+if hasattr(pytest, "global_test_folder"):
+    cache_folder = pytest.global_test_folder / "preprocessing"
+else:
+    cache_folder = Path("cache_folder") / "preprocessing"
+
+set_global_tmp_folder(cache_folder)
```

Comment:
Do we need this? WIll it not get in the way of other tests that are run in the same session?

Reply:
not really! Removed